src/Templates/Home/index.html

```diff
@@ -1,4 +1,4 @@
-{% extends '../Layout/layout.html' %}
+{% extends "../Layout/layout.html" %}
```

Comment:
I don't think this actually matters, but when I was debugging the templates not rendering all the jinja examples used double quotes :shrug: